client/scraper/wikipedia_metadata.py

```diff
@@ -11,13 +10,13 @@ class WikipediaMetadata(Metadata):
     prop_map = {}
 
     def parse_content(self, response):
-        document = lxml.html.fromstring(response.content)
-        title = document.find(".//title").text
+        m = re.search(r"wiki\/(\S+)", response.url)
+        title = m.group(1)
 
         page = wikipedia.page(title)
         self.prop_map["description"] = page.summary
         self.prop_map["title"] = title
```

Comment:
I think that the title should still be something like "Starbucks - Wikipedia the free encyclopedia" or something like that. Maybe we should fetch it from both places?